client/my-sites/comments/main.jsx

```diff
@@ -47,12 +52,33 @@ export class CommentsManagement extends Component {
 			translate,
 		} = this.props;
 
+		const jetpackUpdateLink = `/plugins/jetpack/${ this.props.siteSlug }`;
+
+		const showJetpackUpdateScreen =
+			isJetpack &&
```

Comment:
Do we need both of these Jetpack checks? I'd expect `isJetpackMinimumVersion` to fail on any non-Jetpack sites.

Reply:
@kwight actually we need both of them - `isJetpackMinimumVersion` will fail for non-Jetpack sites (returns `null`), but if we use just that check we'll be showing the nudge on Dotcom sites too.